ebuild-writing/variables/text.xml

```diff
@@ -524,6 +524,23 @@ GLEP 23</uri> for details.
 </body>
 </section>
 
+<section>
+<title>RESTRICT</title>
+<body>
+
+<p>
+While Portage may recognise several different <c>RESTRICT</c> tokens, it is
```

Comment:
I think we use American English in the rest of the manual → recognize.

Reply:
No, we definitely use British English: https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines